e2e/scripts/multipage_apps/pages/06_page_6.py

```diff
@@ -15,3 +15,8 @@
 import streamlit as st
 
 st.header("Page 6")
+
+
+home = st.button("`./streamlit_app.py`")
+if home:
+    st.switch_page("./streamlit_app.py")
```

Comment:
This is weird to me and maybe intended so just asking but shouldn't this be `../streamlit_app.py`?